domains/certificates/Query.API/API.IntegrationTests/Testcontainers/ProjectOriginStack.cs

```diff
@@ -41,7 +41,7 @@ public ProjectOriginStack()
             var hostPort = ((IPEndPoint)udp.Client.LocalEndPoint!).Port;
 
             return new ContainerBuilder()
-                .WithImage("ghcr.io/project-origin/wallet-server:1.3.0")
+                .WithImage("ghcr.io/project-origin/wallet-server:1.4.0")
```

Comment:
Why bump minor in wallet release instead of the patch? Just curious

Reply:
It is a new feature